### PR TITLE
Material Card Fix & Styling

### PR DESCRIPTION
**Pull request by cameronoelsen**
Made a streamlined styling for material design cards that we use on our website condensed into three css classes instead of individualized ones for each page that we used them on. 

They are:
.material-header
.material-info
.material-link

Also cleaned up the Community page with these styling changes to make consistent across pages

### CODE REVIEW

**Comment by cameronoelsen** [comment]
@jhamrick Pinging you on this since you created the Community page! What do you think of this design change?

## Previous Community Page
screen shot 2016-02-17 at 11 32 08 am

## New Styling
screen shot 2016-02-17 at 11 32 15 am

**Comment by jhamrick** [comment]
:+1: looks great!

File: community.html [hunk 1]
```diff
@@ -20,19 +20,24 @@
             <p class="supportparagraph">Please feel free to ask us questions about Project Jupyter!</p>
             <div class="col-md-12">
                 <div class="col-md-4 material material-1">
-                   <p class="community-resource"><a href="http://stackoverflow.com/questions/tagged/jupyter">Jupyter on Stack Overflow</a><p>
+                   <p class="card-header">Jupyter on Stack Overflow<p>
```
Comment (willingc) [inline community.html]: @cameronoelsen Double check the copy pasta on the card-header lines. Looks like some should close with </p>

**Comment by rgbkrk** [comment]
Sweet.

**Pull request merged**